src/components/mobileSample/styles.css

```diff
@@ -0,0 +1,23 @@
+.mobile-container{
+    background-color: var(--color-light-blue);
+    color: var(--color-light-black);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    height: 100vh;
+    width: 50vw;
```

Comment:
Remember to keep alphabetical order on the styles properties so it's easier to find them later.